README.md

```diff
@@ -1,5 +1,9 @@
 # Tower
 
+_NOTE_: Tower's [`master`](https://github.com/tower-rs/tower) branch is currently using [`futures 0.1`](https://docs.rs/futures/0.1.28/futures/), please refer to the [`v0.3.x`](https://github.com/tokio-rs/tokio/tree/v0.1.x) branch to use
```

Comment:
Links to the tokio repo instead of tower XD

Reply:
good catch 😂 I copied the tokio one